src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.tsx

```diff
@@ -69,16 +70,20 @@ const KeyStatTile = ({
               <p className="govuk-body-s">{summary.dataSummary}</p>
             )}
           </div>
-          {summary && summary.dataDefinition && (
-            <Details
-              summary={
-                (summary && summary.dataDefinitionTitle) ||
-                `Define '${config.indicatorLabel}'`
-              }
-            >
-              <div>{summary.dataDefinition}</div>
-            </Details>
-          )}
+          {summary &&
+            summary.dataDefinition &&
+            summary.dataDefinition[0] !== '' && (
```

Comment:
We can use the new elvis operator to write all of this more succinctly as `summary?.dataDefinition?.[0] !== ''`